src/components/dragon/dragon.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable object-curly-newline */
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+import { Badge, Button, Col, Container, Row } from 'react-bootstrap';
+import { useDispatch, useSelector } from 'react-redux';
+import { bookDragons, cancelDragons } from '../../redux/dragon/dragon';
+import './dragon.css';
+
+const DragonPage = () => {
+  const dispatch = useDispatch();
+  const { dragons } = useSelector((state) => state.dragon);
+  const clickHandler = (id, reserved) => {
+    if (reserved === false || reserved === undefined) {
+      dispatch(bookDragons(id));
+    } else {
+      dispatch(cancelDragons(id));
+    }
+  };
+  document.title = "Space Travelers' Hub| Dragons";
+  return (
+    <Container>
+      {dragons.map((dragon) => (
+        <Row className="my-3" key={dragon.id}>
+          <Col md={4}>
+            <img
+              src={
+                dragon.flickr_images.length === 2
+                  ? 'https://live.staticflickr.com/8578/16655995541_7817565ea9_k.jpg'
+                  : dragon.flickr_images
+              }
+              alt="dragon"
+              className="img-fluid"
+            />
+          </Col>
+          <Col md={8}>
+            <h3>{dragon.name}</h3>
+            <p>
+              <Badge bg="primary">{dragon.reserved ? 'Reserved' : ''}</Badge>{' '}
```

Comment:
- [ ] [OPTIONAL] As required by project requirements. It is suggested that you make use of React conditional rendering syntax   as used in the Missions and Rockets components


## EXAMPLE
```
        {mission.reserved && (
          <div className="w-64 flex justify-around text-xs">
            <button
              type="button"
              className="text-sm text-red-600 border-red-400 rounded-sm border-2 px-1 py-0.5"
              onClick={() => dispatch(leaveMission(mission.mission_id))}
            >
              {' '}
              leave mission
            </button>
          </div>
        )}
        {!mission.reserved && (
          <div className="w-64 flex justify-around text-xs">
            <button
              type="button"
              className="text-sm text-gray-600 border-gray-400 rounded-sm border-2 px-1 py-0.5"
              onClick={() => dispatch(joinMission(mission.mission_id))}
            >
              {' '}
              Join Mission
            </button>
          </div>
        )}
```

The above is an example and may need to be adapted to your project.